--- rubygem-diff-lcs-1.1.2/metadata.tag~	2012-02-17 01:47:27.217570088 +0100
+++ rubygem-diff-lcs-1.1.2/metadata	2012-02-17 01:47:42.264584304 +0100
@@ -8,7 +8,8 @@ date: 2004-10-20
 summary: Provides a list of changes that represent the difference between two sequenced collections.
 require_paths: 
   - lib
-author: Austin Ziegler
+authors:
+- Austin Ziegler
 email: diff-lcs@halostatue.ca
 homepage: http://rubyforge.org/projects/ruwiki/
 rubyforge_project: ruwiki
@@ -24,7 +25,19 @@ required_ruby_version: !ruby/object:Gem:
       - !ruby/object:Gem::Version 
         version: 1.8.1
   version: 
+required_rubygems_version: !ruby/object:Gem::Requirement 
+  none: false
+  requirements: 
+  - - ">="
+    - !ruby/object:Gem::Version 
+      hash: 3
+      segments: 
+      - 0
+      version: "0"
+
 platform: ruby
+licenses: []
+cert_chain: []
 files: 
   - bin
   - ChangeLog
